main.js

```diff
@@ -0,0 +1,236 @@
+    $('body').attr('class', classValue);
+
+  };
+
+  // Update the current weather conditions in our weather data structure.
+  let addCurrentData = function (data) {
+    weather.currentWeather = {};
+    let formattedConditions = capitalizeConditionsString(data.weather[0].description);
+
+    // Setup the object containing information with the current weather conditions at the time of search.
+    let currentWeather = {
+      temperature: data.main.temp.toFixed(),
+      city: data.name,
+      conditions: formattedConditions,
+      icon: data.weather[0].icon
+    };
+
+    weather.currentWeather = currentWeather;
+  };
+
+  // Update the forecasted weather conditions in our weather data structure.
+  // Arbitrarily, the function filters out the forecast data and picks its 5 results timestamped at 12 p.m. or 1 p.m.
+  let addForecastData = function (data) {
+    weather.fiveDayForecast = [];
+
+    // Cycle through the array of forecast data
+    data.list.forEach( function (result) {
+      let day = {};
+      // Check for results around noon to be added to weather array
+      if (moment.unix(result.dt).format('HH') == 12 || moment.unix  (result.dt).format('HH') == 13) {
+
+        // Get the formatted conditions, temperature, and day of the week
+        let formattedConditions = capitalizeConditionsString(result.weather[0].description);
+
+        let day = {
+          conditions: formattedConditions,
+          temperature: result.main.temp.toFixed(),
+          dayOfWeek: moment.unix(result.dt).format('dddd'),
+          icon: result.weather[0].icon
+        };
+
+        // Push each day object to weather.fiveDayForecast
+        weather.fiveDayForecast.push(day);
+      }
+    });
+    // Render all of the new weather results
+    renderWeather();
+  };
+
+  // Get the user's current location if possible and search for the weather at his/her location.
+  // The function will alert if it's unable to get the location.
+  let useCurrentLocation = function () {
+    // Check that geolocation is available in the client browser
+    if (!navigator.geolocation) {
+      alert('Cannot get your current location!');
+      return;
+    }
+
+    function success(position) {
+      let latitude  = position.coords.latitude;
+      let longitude = position.coords.longitude;
+      fetchCurrent(latitude, longitude);
+      fetchForecast(latitude, longitude);
+    }
+
+    function error() {
+      alert('Unable to retrieve your location');
+    }
+
+    navigator.geolocation.getCurrentPosition(success, error);
+  };
+
+  // API call to get current weather data from OpenWeatherMap based on the user's query.
+  let fetchCurrent = function (query) {
+
+    let formattedQueryCurrent = 'http://api.openweathermap.org/data/2.5/weather?q=' + query + ',us&units=imperial&appid=0b3fcf70b1228659daf9b5a171c0e96f';
```

Comment:
Could you have abstracted this into a function?  This routine is too similar to below.